### ISSUE TITLE
infra: add metrics

### ISSUE THREAD

**Issue by agouaillard** [agouaillard/Large-Long-Lived-Connections-Challenge#9]
depends on #6 .

Modified web app code and ws-server code to print more info on current usage to start benchmarking.
Eventually, it would be nice to have the web app display that in the HTML, and not through the console.

**Comment by agouaillard** [comment]
can always add more, and those are per node process, but it will do for now.
